src/Security/SslContext.cs

```diff
@@ -523,7 +523,7 @@ public SslStatus SetEncryptionCertificate (SecIdentity identify, IEnumerable<Sec
 			get {
 				IntPtr value;
 				result = SSLCopyPeerTrust (Handle, out value);
-				return (value == IntPtr.Zero) ? null : new SecTrust (value);
+				return (value == IntPtr.Zero) ? null : new SecTrust (value, true);
```

Comment:
The second param to SecTrust (handle, true) is transferring ownership to SecTrust, so it knows to dispose the handle when it's done. The default is false, which is why this was a leak.